source/includes/markdown/_13-platform-ui.html.md

```diff
@@ -306,6 +306,77 @@ Before you begin, you'll need a developer sandbox in order to use App Components
 <br>
 That's it! At this point, feel free to keep exploring how changes in the server affects data in the task's Widget. Once you're ready to define an app, [click here to create your own app](/docs/configurations) with App Components.
 
+# Installing an App
+
+<span class="beta-indicator">BETA</span> - For access, please see [Overview of App Components](/docs/overview-of-app-components)
+
+When a user connects an app with App Components to Asana for the first time, they will go through an installation flow. This involves walking the user through the app's features (i.e., "value props"), authorizing the app, and adding it to projects in Asana. The installation flow can be triggered through either one of two ways:
```

Comment:
consider: do you think it's worth calling out explicitly that subsequent interactions with the same app by the same user won't trigger this installation flow?

consider: we have a deeplink to this installation flow that works regardless if a user has gone through it before. It looks like https://app.asana.com/-/install_platform_ui_app?app_id=<app_client_id> . Should we publicize this here?

Reply:
Definitely some helpful information (that would even benefit the QA process). I'll make sure it's in our content